src/interface/src/app/map/map-control-panel/map-control-panel.component.html

```diff
@@ -1,26 +1,9 @@
 <div class="controls-container">
 
-  <div *ngIf="mapHasDataLayer" class="legend-container">
-    <h1><b>Map {{selectedMap?.name}}</b> Legend</h1>
-
-    <div class="legend-box">
-      <!-- Name of the kind of data -->
-      <h3>{{selectedMap?.config?.dataLayerConfig?.legend_name}}</h3>
-
-      <div class="legend-title">
-        <h2>{{selectedMap?.config?.dataLayerConfig?.display_name}}</h2>
-        <button mat-icon-button
-          [matMenuTriggerFor]="popoverMenu">
-          <mat-icon color="primary">info_outline</mat-icon>
-        </button>
-        <mat-menu #popoverMenu="matMenu">
-          <app-layer-info-card [dataLayerConfig]="selectedMap?.config?.dataLayerConfig"></app-layer-info-card>
-        </mat-menu>
-      </div>
-
-      <!-- Legend for the currently shown map -->
-      <app-legend [legend]="getSelectedLegend()"></app-legend>
+  <div *ngIf="mapHasDataLayer" class="title-container">
+    <h1><b>Map {{selectedMap?.name}}</b></h1>
```

Comment:
Nit: I'd add a colon after Map here since "Map" is now sort of a prefixing title for the map name, whereas before the Legend was the actual "noun".

Reply:
Actually, do you think we should have that header at all? Looking at it now, it kind of looks like it's saying the transparency bar is for that specific map. 
<img width="410" alt="Screenshot 2023-07-12 at 11 11 51 AM" src="https://github.com/OurPlanscape/Planscape/assets/18537927/12c4a862-3e5d-4198-90e7-7fc6fe401224">

Reply:
This is what it looks like without the header
<img width="548" alt="Screenshot 2023-07-12 at 11 23 30 AM" src="https://github.com/OurPlanscape/Planscape/assets/18537927/08e50eb2-77a6-4135-b37a-a2caa06ddb07">

Reply:
(For documentation's sake): Checked with Scott and going forward with 'Map Control Panel' header and updated transparency slider label 
<img width="567" alt="Screenshot 2023-07-12 at 11 53 52 AM" src="https://github.com/OurPlanscape/Planscape/assets/18537927/da693abe-da06-416e-9620-1abebd32fe81">